README.md

```diff
@@ -1,13 +1,18 @@
 # Project Auth
+This project was made by Malin Göthe and Felicia Liabäck Löwstedt.
+This week we have learned to authenticate users and handle passwords.
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+- How to authenticate users using tokens
+- How to securely store passwords in your databases
+- How to think about security and defensive design when building frontend or backend code
+- Some common attacks which are used against sites and how to protect against them
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Our project consists of two parts, a backend API, and a React frontend. In the backend we have a user model and thoughts model using mongoose with properties for the registered user, and to store a user's access token. In frontend we have a registration form that POST to our API. The passwords in the database are encrypted with bcrypt. 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-auth-mg-fll.netlify.app/login
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```

![image](https://user-images.githubusercontent.com/50926159/120894769-7340ad80-c61a-11eb-872e-4d33022a626e.png)